packages/frontend/src/components/tags.tsx

```diff
@@ -2,21 +2,26 @@ import { Sport } from "@esp-group-one/types";
 
 interface Tagging {
   sportName: Sport;
+  active: boolean;
```

Comment:
Active should be optional which defaults to true
```suggestion
  active?: boolean;
```